test/integration/scheduler_perf/util.go

```diff
@@ -44,7 +56,7 @@ const enableEquivalenceCache = true
 // remove resources after finished.
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
-func mustSetupScheduler() (schedulerConfigurator scheduler.Configurator, destroyFunc func()) {
+func mustSetupScheduler() (configurator scheduler.Configurator, destroyFunc func()) {
```

Comment:
I was planning to take this function out, as scheduler dir in integration tests has already a utility file which creates an API server + scheduler.

Reply:
Sure, where do you think this function belongs to, the `integration/utils.go`?

Reply:
Yes. We can import functions from `integration/utils.go`